hat/app/org/hatdex/hat/phata/models/HatClaims.scala

```diff
@@ -29,7 +29,10 @@ import play.api.libs.json.{ Json, OFormat, Reads, Writes }
 case class ApiVerificationRequest(
     applicationId: String,
     email: String,
-    redirectUri: String)
+    redirectUri: String) {
+  override def toString() =
+    s"applicationId: ${applicationId}, email:REDACTED, redirectUri: ${redirectUri}"
```

Comment:
I would add the class name back in (the way a case class does) i.e.:
`ApiVerificationRequest(applicationId:...)`
then IntelliJ gives a warning for templated strings where the curly braces are not required so `$applicationId` instead of `${applicationId}` otherwise our code is littered with warnings all over the place

Reply:
I did not know that IntelliJ did that, seems helpful.